pkgs/applications/misc/rofi-bluetooth/default.nix

```diff
@@ -26,7 +27,10 @@ stdenv.mkDerivation rec {
     install -D --target-directory=$out/bin/ ./rofi-bluetooth
 
     wrapProgram $out/bin/rofi-bluetooth \
-      --prefix PATH ":" ${lib.makeBinPath [ rofi-unwrapped bluez ] }
+      --prefix PATH ":" ${lib.makeBinPath [
+        (if withWayland then rofi-wayland else rofi)
```

Comment:
I think it would be easier to just rely on PATH. Also rebuilding the package to just change the wrapper is rather wasteful.

Reply:
I'm sorry, but I don't quite understand what you mean to just rely on PATH. Do mean to not include a rofi version in the package at all and just rely on the user to add a rofi to path?

Reply:
Yes, especially that we also need an option Incase Wayland is used which we generally try to avoid.